src/utils/hooks/useTriggerProcessing.tsx

```diff
@@ -1,16 +1,41 @@
 import { useState, useEffect } from 'react'
 import { useGetTriggersQuery } from '../../utils/generated/graphql'
 
-type triggerTypes = 'applicationTrigger' | 'reviewTrigger'
+type TriggerType = 'applicationTrigger' | 'reviewAssignmentTrigger' | 'reviewTrigger'
```

Comment:
Any reason not to use the Graphql type here?

Reply:
We don't have a GraphQL type for this. We have a type for the individual triggers (`onApplicationCreate`, etc.), but not on a per-table basis.